udev/readme.md

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
Could you rename it to `README.md`, and update the main readme file to point to here? (It already mentions udev; no need to have it in two places.)